src/apis/downloadPdf.js

```diff
@@ -1,16 +1,37 @@
 /**
  * Downloads the pdf document with or without annotations added by WebViewer UI.
  * @method WebViewerInstance#downloadPdf
- * @param {boolean} [includeAnnotations=true] Whether or not to include annotations added by WebViewer UI.
+ * @param {object} [options] An object that contains the following download options.
+ * @param {string} [options.filename] The filename of the downloaded document.
+ * @param {string} [options.downloadType='pdf'] The type to download the file as, by default this is "pdf". PDF and image files can only be downloaded as PDFs, but office files can be downloaded as "pdf" or as "office" if you want to get the original file without annotations.
```

Comment:
`"office" if you want to get the original file without annotations.`
slightly misleading?

I think if we had a pdf file that already had annots and set it  download type to office, i think the annots are still in there...?

Reply:
idk haha I just copied the document from `getFileData` over here. How do you think we should explain this? I don't really know how this works haha.

Reply:
nevermind i misread it.
if you put download type as 'office' for a pdf file, it will still have its original annots.
but if its an office file, it wont have annots. so this comment is correct.